src/components/views/Instance/List.tsx

```diff
@@ -99,7 +101,7 @@ class InstanceList extends React.Component<IProps, IState> {
                 const modifiedinstance = instance as Instance;
                 if (!enableVersionWorkaround) {
                     modifiedinstance.canAccess = modifiedinstance.online
-                        ? modifiedinstance.accessible
+                        ? !!modifiedinstance.accessible
```

Comment:
You're making my IDE very angry because this is a boolean
```suggestion
                        ? modifiedinstance.accessible
```